rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -212,33 +209,72 @@ def _lm_specific_token_cleanup(
 
         Returns: Cleaned up token ids and token strings.
         """
-        from rasa.nlu.utils.hugging_face.registry import model_tokens_cleaners
 
-        return model_tokens_cleaners[self.model_name](split_token_ids, token_strings)
+        def _remove_prefixes(token: str) -> str:
+            """Remove the tokenizer-specific prefix.
+
+            BERT tokenizers are treated separately since their prefix signifies a
+            sub-token and the builtin `convert_tokens_to_string` only works for a
+            list of tokens, while we apply it to an individual token. For other
+            tokenizers the prefixes signify whitespace or end-of-words.
+            """
+            if issubclass(
+                type(self.tokenizer), transformers.tokenization_bert.BertTokenizer
+            ) or issubclass(
+                type(self.tokenizer), transformers.tokenization_bert.BertTokenizerFast
+            ):
+                return token.replace("##", "")
```

Comment:
I think we should maybe make this a constant. I know we only need it here in this specific case, but I think it'll help document... really hated that HF doesn't have some sort of constant I could search for :)